### DISCUSSION TITLE
Does Homespan work with Ios 18 ?

### DISCUSSION THREAD

**Discussion by Sill1000** [HomeSpan/HomeSpan#932] [General]
Hello,

My HA was working perfectly with IOS17. Upgraded to IOS18 a lot of sensors are disconnected.
Tryed to reset the HomePod, reset all my Homespan devices. But now I canot pair it again !! 

```
Trying to connect to .... Waiting 2s
E (15491) wifi:sta is connecting, return error
Trying to connect to .... Waiting 4s
Trying to connect to .... Waiting 8s
```

Pinging the ESP get:

imac2:~ $ ping xxx.xxx.x.xxx
PING xxx.xxx.x.xxx (xxx.xxx.x.xxx): 56 data bytes
Request timeout for icmp_seq 0
Request timeout for icmp_seq 1
Request timeout for icmp_seq 2
Request timeout for icmp_seq 3
Request timeout for icmp_seq 4
64 bytes from xxx.xxx.x.xxx: icmp_seq=2 ttl=255 time=3224.508 ms
64 bytes from xxx.xxx.x.xxx: icmp_seq=3 ttl=255 time=2227.145 ms
64 bytes from xxx.xxx.x.xxx: icmp_seq=4 ttl=255 time=1230.675 ms
Request timeout for icmp_seq 8
Request timeout for icmp_seq 9
Request timeout for icmp_seq 10
Request timeout for icmp_seq 11
64 bytes from xxx.xxx.x.xxx: icmp_seq=9 ttl=255 time=3217.220 ms
64 bytes from xxx.xxx.x.xxx: icmp_seq=10 ttl=255 time=2213.920 ms
64 bytes from xxx.xxx.x.xxx: icmp_seq=11 ttl=255 time=1217.067 ms
64 bytes from xxx.xxx.x.xxx: icmp_seq=12 ttl=255 time=221.246 ms
64 bytes from xxx.xxx.x.xxx: icmp_seq=15 ttl=255 time=1206.802 ms
64 bytes from xxx.xxx.x.xxx: icmp_seq=16 ttl=255 time=212.034 ms

Why those strange delay ????

What to do ?

Thanks

**Comment by frankonski** [comment]
No issues with iOS 18 from my end, with one iPhone and one iPad upgraded. I don’t know if any of my hubs / AppleTV got any update recently though..

The latency in your pings is all over the place.

**Comment by Sill1000** [comment]
Thanks !!! Back to normal after rebooting all switches !
